prisma/schema.prisma

```diff
@@ -62,3 +69,56 @@ model VerificationToken {
 
     @@unique([identifier, token])
 }
+
+model Trips {
+    id                  String      @id @default(cuid())
+    distanceTraveled    DateTime
```

Comment:
Why is distanceTraveled DateTime? Wouldn't it be BigInt or Decimal?

Reply:
And that's why we do reviews, lol. Oversight on my part, I'll fix it to Decimal when I get back